shell/platform/embedder/embedder.h

```diff
@@ -1118,9 +1121,82 @@ typedef struct {
   const char* tooltip;
 } FlutterSemanticsNode;
 
+/// A node that represents some semantic data.
+///
+/// The semantics tree is maintained during the semantics phase of the pipeline
+/// (i.e., during PipelineOwner.flushSemantics), which happens after
+/// compositing. Updates are then pushed to embedders via the registered
+/// `FlutterUpdateSemanticsCallback2`.
```

Comment:
I don't think we've referred directly to framework code from here before, but I can't think of many reasons not to:

```suggestion
/// `FlutterUpdateSemanticsCallback2`.
///
/// @see https://api.flutter.dev/flutter/semantics/SemanticsNode-class.html
```

If we ever change our domain name we'll need to fix this, but that's happened only once. We could/should probably eventually add some form of URL liveness checking for this.

Reply:
Updated!